side.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Introduction</title>
+<link rel="icon" href="favicon.ico" type="image/vnd.microsoft.icon">
+<link rel="shortcut icon" href="favicon.ico" type="image/vnd.microsoft.icon">
+<link rel=stylesheet href=se.css>
+<link rel=prev href=quick.html title="Quick Tour">
+<link rel=next href=install.html title=Install>
+<script src=docs.js></script>
+
+<h1>Selenium-IDE</h1>
+
+<h3>Introduction</h3>
+
+<p>The <em>Selenium-IDE(Integrated Development Environment) </em>is the tool you use to develop your
+    Selenium test cases. It’s an easy-to-use Chrome and Firefox extension and is generally the most
```

Comment:
It's an easy-to-use Web Extension, usable in Chrome, Firefox, and any browser that supports that standard.